zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -64,11 +65,12 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.collect.Sets;
```

Comment:
tbh we keep moving these imports in various PRs. should we just adopt a style that could work consistently in IDE?

Reply:
Yes, for that kind of code style issue, I've talked with @zjffdu and concluded not to change this time. :-) Thanks for letting me know that.

Reply:
yes, but we just need to all use the same style, then it will stop moving

Reply:
You’re right. Someday we need to adopt same style at a time. And I’m worried about that code style issue always become low priority because it affected a lot but there’s no new feature. It’s a bit hard to make consensus for that reason. :-)